lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_version_helper.rb

```diff
@@ -1,18 +1,40 @@
 module Fastlane
     module Helpers
+      # A module containing helper methods to manipulate/extract/bump iOS version strings in xcconfig files
+      #
       module IosVersionHelper
+        # @!group The indices for various parts of the version parts array.
+        # The index for the major version number part
         MAJOR_NUMBER = 0
+        # The index for the minor version number part
         MINOR_NUMBER = 1
+        # The index for the hotfix version number part
         HOTFIX_NUMBER = 2
+        # The index for the build version number part
         BUILD_NUMBER = 3
+        # @!endgroup
   
+        # Returns the public-facing version string.
+        #
+        # @return [String] The public-facing version number, extracted from the VERSION_LONG entry of the xcconfig file.
+        #         - If this version is a hotfix (more than 2 parts and 3rd part is non-zero), returns the "X.Y.Z" formatted string
+        #         - Otherwise (not a hotfix / 3rd part of version is 0), returns "X.Y" formatted version number
+        #
         def self.get_public_version
           version = get_build_version
           vp = get_version_parts(version)
-          return "#{vp[MAJOR_NUMBER]}.#{vp[MINOR_NUMBER]}" unless is_hotfix(version)
+          return "#{vp[MAJOR_NUMBER]}.#{vp[MINOR_NUMBER]}" unless is_hotfix?(version)
           "#{vp[MAJOR_NUMBER]}.#{vp[MINOR_NUMBER]}.#{vp[HOTFIX_NUMBER]}"
         end
   
+        # Compute the name of the next release version.
+        #
+        # @param [String] version The current version that we want to increment
+        #
+        # @return [String] The predicted next version, in the form of "X.Y".
+        #         Corresponds to incrementing the minor part, except if it reached 10
+        #         (in that case we go to the next major version, as decided in our versioning conventions)
```

Comment:
I remember searching for a link to these conventions but without luck. The best I could come up with was [the WordPress versions page](https://codex.wordpress.org/WordPress_Versions).